### DISCUSSION TITLE
Questions about the submit

### DISCUSSION THREAD

**Discussion by hongyingyue** [PaddlePaddle/PaddleSpatial#43] [BaiduKDDCup2022] [answered]
Thanks for hosting an interesting competition. I have some questions that need your kindly answer

1. About the submission
Could you please provide a template about how to submit to get the result. From current version, I guess we can select paddle/tf/torch as different platforms, how about its version? how to assign other library and its version?

2. About the test data
Could you please provide more detailed information about test dataset. I just saw there will be a gap between train data and test data. Will be there also a gap between the Phase 1 test data and Phase 2 test data?  So the gap will be unknown ?

Thanks

**Comment by zhoujingbo** [comment, marked as answer]
> Thanks for hosting an interesting competition. I have some questions that need your kindly answer
> 
> 1. About the submission
>    Could you please provide a template about how to submit to get the result. From current version, I guess we can select paddle/tf/torch as different platforms, how about its version? how to assign other library and its version?

A guideline for submitting the model can be found here:https://github.com/PaddlePaddle/PaddleSpatial/tree/main/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation (it is still in process)
Thanks a lot.


> 2. About the test data
>    Could you please provide more detailed information about test dataset. I just saw there will be a gap between train data and test data. Will be there also a gap between the Phase 1 test data and Phase 2 test data?  So the gap will be unknown ?

Yes, and the Gap between Phase 1 and Phase 2 is unknown. Thanks

> 
> Thanks
